cmd/restic/cmd_rewrite.go

```diff
@@ -35,6 +35,9 @@ Please note that the --forget option only removes the snapshots and not the actu
 data stored in the repository. In order to delete the no longer referenced data,
 use the "prune" command.
 
+The option --snapshot-summary [-s] creates a new snapshot with snapshot summary data attached.
```

Comment:
Let's just describe the long option name here. Please also extend the description to clarify that the snapshot summary is only calculated if it doesn't exist.